src/MessageHandler/DatasetSubmissionFilerHandler.php

```diff
@@ -153,7 +153,7 @@ private function processFile(File $file): void
         // TODO implement method
 
         try {
-            $newFileDestination = $this->datastore->addFile(fopen($filepath, 'r'));
+            $newFileDestination = $this->datastore->addFile(['fileStream' => fopen($filepath, 'r')]);
```

Comment:
Should we turn this into a helper class so it's a little easier?

Reply:
Even if its a helper class, the object of the helper class should return resource type inside an array